runtime/src/iree/hal/executable_layout.c

```diff
@@ -13,6 +13,38 @@
 #include "iree/hal/device.h"
 #include "iree/hal/resource.h"
 
+//===----------------------------------------------------------------------===//
+// iree_hal_descriptor_set_layout_t
+//===----------------------------------------------------------------------===//
+
+#define _VTABLE_DISPATCH(descriptor_set_layout, method_name) \
+  IREE_HAL_VTABLE_DISPATCH(descriptor_set_layout,            \
+                           iree_hal_descriptor_set_layout, method_name)
```

Comment:
https://github.com/iree-org/iree/runs/7957865736?check_suite_focus=true

```
/home/runner/actions-runner/_work/iree/iree/runtime/src/iree/hal/executable_layout.c:48:9: error: '_VTABLE_DISPATCH' macro redefined [-Werror,-Wmacro-redefined]
#define _VTABLE_DISPATCH(executable_layout, method_name)                  \
        ^
/home/runner/actions-runner/_work/iree/iree/runtime/src/iree/hal/executable_layout.c:20:9: note: previous definition is here
#define _VTABLE_DISPATCH(descriptor_set_layout, method_name) \
        ^
```

Reply:
weird that msvc doesn't care